src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/CreateIssueAction.kt

```diff
@@ -31,8 +31,9 @@ class CreateIssueAction(
                 .waitForDialog()
                 .showAllFields()
                 .selectProject(project.name)
-                .selectIssueType { issueTypes -> seededRandom.pick(issueTypes)!! }
-                .fill("summary", "This is a simple summary")
+                .selectIssueType {
+                    issueTypes -> seededRandom.pick(issueTypes.filter { it != "Epic" } )!!
```

Comment:
To be clear: Epic's are not special in general.
Our datasets are misconfigured. The create issue screen doesn't show the required Epic Name field:
![image](https://user-images.githubusercontent.com/1194704/74131754-58585780-4be5-11ea-9c33-b89fffc13610.png)